### PR TITLE
fix: product feature library unit test failing (#11384)

### LINKED ISSUE #11384: Fix pipeline on product library unit tests

### PR DESCRIPTION
**Pull request by mszczepaniak**
Closes #11384

### CODE REVIEW

File: feature-libs/product/bulk-pricing/components/bulk-pricing-table/bulk-pricing-table.component.spec.ts [hunk 1]
```diff
@@ -27,6 +29,60 @@ const mockTierWithoutMaxQuantity: BulkPrice = {
...
+    minQuantity: 100,
+    priceType: 'BUY',
+    value: 2.99,
+    formattedDiscount: '-25%',
+    discount: 25,
+  },
+];
+
 class MockRoutingService implements Partial<RoutingService> {
   go() {}
```
Comment (marlass) [inline feature-libs/product/bulk-pricing/components/bulk-pricing-table/bulk-pricing-table.component.spec.ts]: It is not used. Can we remove?

File: feature-libs/product/bulk-pricing/components/bulk-pricing-table/bulk-pricing-table.component.spec.ts [hunk 2]
```diff
@@ -35,8 +91,10 @@ class MockRoutingService implements Partial<RoutingService> {
 }
 
 class MockBulkPricingService implements Partial<BulkPricingService> {
+  PRODUCT_SCOPE = 'bulkPrices';
```
Comment (marlass) [inline feature-libs/product/bulk-pricing/components/bulk-pricing-table/bulk-pricing-table.component.spec.ts]: Can we remove that as well?

**Review by marlass — CHANGES_REQUESTED** [review]

File: feature-libs/product/bulk-pricing/components/bulk-pricing-table/bulk-pricing-table.component.spec.ts (hunk 2 above)
Comment (mszczepaniak) [reply feature-libs/product/bulk-pricing/components/bulk-pricing-table/bulk-pricing-table.component.spec.ts]: yes

File: feature-libs/product/bulk-pricing/components/bulk-pricing-table/bulk-pricing-table.component.spec.ts (hunk 1 above)
Comment (mszczepaniak) [reply feature-libs/product/bulk-pricing/components/bulk-pricing-table/bulk-pricing-table.component.spec.ts]: yes

**Review by marlass — APPROVED** [review]

**Pull request merged**